Project.toml

```
@@ -10,7 +10,7 @@ Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 URIParser = "30578b45-9adc-5946-b283-645ec420af67"
 
 [compat]
-FilePathsBase = "0.6"
+FilePathsBase = ">= 0.6"
```

Lower-bound only version bounds are a problem. Basically don't use them.
It means the package manager things this release is compatible with Everything `>=0.6` including v2, v3 etc.
So it will downgrade this package later (once we only support `v1.0`) to this old version because it thinkts that it supports `v2` for example
```suggestion
FilePathsBase = "0.6, 0.7, 0.9"
```

---

Good suggestion, thank you!

---

Is there a reason you excluded 0.8?